sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/__init__.py

```diff
@@ -114,6 +114,7 @@
     "ClassifyDocumentResult",
     "ClassificationCategory",
     "AnalyzeHealthcareEntitiesAction",
+    "TextAnalyticsLROPoller",
```

Comment:
do we really want to expose the poller to users?, i.e. do we want them to import the poller from here?

I guess maybe for `from_continuation_token`?

Reply:
It's more to expose for documentation, so that users can see the metadata and (eventual) cancel method present on the poller. I wouldn't expect anyone to import this directly into their code.